core/includes/bootstrap.inc

```diff
@@ -2321,12 +2321,22 @@ function watchdog($type, $message, $variables = array(), $severity = WATCHDOG_NO
  *   (optional) The translated message to be displayed to the user. For
  *   consistency with other messages, it should begin with a capital letter and
  *   end with a period.
+ * @param string $message
+ *   (optional) The translated message to be displayed to the user. For
+ *   consistency with other messages, it should begin with a capital letter and
+ *   end with a period.
  * @param string $type
- *   (optional) The message's type. Defaults to 'status'. These values are
+ *   (optional) The type of the message. Defaults to 'status'. These values are
```

Comment:
Not sure about informal tone in documentation. I personnaly don't mind, but we should be consistant across the whole documentation.